distro_data/ZOS_Software_List.json

```diff
@@ -185,7 +185,7 @@
     },
     {
         "packageName":"Unzip",
-        "description":"RocketSoftware hosted UnZip is an extraction utility for archives compressed in .zip format. Unzip is also compatible with PKZIP and PKUNZIP utilities for MS-DOS as well as Info-ZIP�s and supports bzip2 compression methods as well.",
+        "description":"RocketSoftware hosted UnZip is an extraction utility for archives compressed in .zip format. Unzip is also compatible with PKZIP and PKUNZIP utilities for MS-DOS as well as Info-ZIP's and supports bzip2 compression methods as well.",
```

Comment:
I just tried all your changes...the ' here is actually adding string error, since it's beginning a new quoted string without another closing '
Same case with ` too. So I just removed the character

Reply:
Are you suggesting some changes in the PR?

Reply:
I miss this. This is definitely something we'll have to keep an eye on with these descriptions.

Reply:
Noted